doc/source/whatsnew/v1.0.0.rst

```diff
@@ -260,6 +260,31 @@ The following methods now also correctly output values for unobserved categories
    df.groupby(["cat_1", "cat_2"], observed=False)["value"].count()
 
 
+Avoid iterating over ``Series.str``
```

Comment:
So this is a very niche / questionable application to begin with, so probably don’t need a dedicated section for this. Can you just make this a one-liner in the `Deprecations` section that Series.str.__iter__ has been deprecated and will be removed in a future version?

Reply:
Sure, no problem